open-metadata-implementation/integration-services/organization-integrator/build.gradle

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright Contributors to the ODPi Egeria project.
+ */
+
+
+dependencies {
+    implementation project(':open-metadata-implementation:frameworks:audit-log-framework')
+    implementation project(':open-metadata-implementation:frameworks:open-connector-framework')
+    implementation project(':open-metadata-implementation:repository-services:repository-services-apis')
+    implementation project(':open-metadata-implementation:governance-servers:integration-daemon-services:integration-daemon-services-api')
+    implementation project(':open-metadata-implementation:repository-services:repository-services-apis')
+    implementation project(':open-metadata-implementation:access-services:community-profile:community-profile-api')
```

Comment:
Where we don't have anything to build, we often don't even have a build.gradle, nor do we need the dependencies for submodules as this is driven by the overall settings.gradle
Should be harmless, though inconsistent with what we do around for example OMASs